### PR TITLE
add a new header

### PR DESCRIPTION
**Pull request by XxKikoxX**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.body {
+  display: sticky;
```
Comment (liebeckm) [inline css/styles.css]: better to put the sticky property to the header, otherwise our body won't scroll?

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  top: 0;
+  flex-direction: column;
+  background-color: rgb(78, 82, 83);
+}
+.header {
```
Comment (liebeckm) [inline css/styles.css:5]: for header, you don't need to make a new class. you can use the existing one <header>.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Quizz-App</title>
+  </head>
+  <header class="header">
```
Comment (liebeckm) [inline index.html:10]: see above; for header, you don't need to make a new class.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <h1 class="h1">Quiz App</h1>
+  </header>
+  <body class="body">
+    <p class="body-text">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos tempora
+      iusto cum, molestiae temporibus quod aut tempore eveniet nam qui assumenda
+      soluta ad doloribus sit libero quisquam dignissimos reprehenderit officia!
+    </p>
+    <p class="body-text">
```
Comment (liebeckm) [inline index.html:19]: a little complicated with the classes. you. could refer to body only or, if you want to test different classes, invent new classes, e.g. class="body-text-01" or class="body-text-02"

File: css/styles.css (hunk 1 above)
Comment (XxKikoxX) [reply css/styles.css]: thank you

**Review by liebeckm — COMMENTED** [review]
see comments

**Pull request merged**